Packs/Netskope/Integrations/NetskopeEventCollector/README.md

```diff
@@ -8,29 +8,31 @@
     | --- | --- | --- |
     | Server URL |  | True |
     | API token |  | True |
-    | API Version | The API version to use \(v1 or v2\). | False |
     | Trust any certificate (not secure) |  | False |
     | Use system proxy settings |  | False |
-    | First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) |  | False |
-    | Max events per fetch | The maximum amount of events to retrieve \(up to 30000 events\). | False |
+    | Max events per fetch | The maximum amount of events to retrieve per each event type. For more information about event types see the help section. | False |
```

Comment:
Don't you want to add the limitations also to the readme? 
Under the known limitation section?

Reply:
WDYM? I have this https://github.com/demisto/content/blob/c9a58b26fef538397c2572b9836c04ba1f719846/Packs/Netskope/Integrations/NetskopeEventCollector/README.md#fetch-events-limitation

Reply:
My bad - Missed that.
Do we want to add also the API limitation regarding indexation?

Reply:
Waiting for the Netskope team investigation resolution on this